src/Test/MakerTestEnvironment.php

```diff
@@ -259,37 +259,37 @@ public function getGeneratedFilesFromOutputText()
         return array_map('trim', $matches[3]);
     }
 
-    public function fileExists(string $file)
+    public function fileExists(string $file): bool
     {
         return $this->fs->exists($this->path.'/'.$file);
     }
 
-    public function runPhpCSFixer(string $file)
+    public function runPhpCSFixer(string $file): MakerTestProcess
     {
         return MakerTestProcess::create(sprintf('php vendor/bin/php-cs-fixer --config=%s fix --dry-run --diff %s', __DIR__.'/../Resources/test/.php_cs.test', $this->path.'/'.$file), $this->rootPath)
                                ->run(true);
     }
 
-    public function runTwigCSLint(string $file)
+    public function runTwigCSLint(string $file): MakerTestProcess
     {
         return MakerTestProcess::create(sprintf('php vendor/bin/twigcs --config ./.twig_cs.dist %s', $this->path.'/'.$file), $this->rootPath)
                                ->run(true);
     }
 
-    public function runInternalTests()
+    public function runInternalTests(): ?MakerTestProcess
     {
         $finder = new Finder();
         $finder->in($this->path.'/tests')->files();
         if ($finder->count() > 0) {
             // execute the tests that were moved into the project!
-            return MakerTestProcess::create(sprintf('php %s', $this->rootPath.'/vendor/bin/simple-phpunit'), $this->path)
+            return MakerTestProcess::create(sprintf('php %s', $this->path.'/bin/phpunit'), $this->path)
```

Comment:
This changes needs extra scrutiny - test time is cut in half (usually an indicator im doing something wrong) but I'm not seeing an actual problem. Before we were calling `/maker-bundle/vendor/bin/simple-phpunit /maker-bundle/cache/generated-project`. This PR calls the generated projects phpunit directly e.g. `/maker-bundle/cache/generated-project/bin/phpunit`

Reply:
I think (?) I originally did this... to avoid PHPUnit being downloaded over and over again inside *every* generated project. I figured: PHPUnit is already there or Maker, so let's just use it! So, I don't see any problem with this.

As far as why the test time is cut in half, that is interesting. I believe you already told me on Slack that the number of tests before and after are identical - so we didn't somehow "lose" some tests. One theory is that my plan backfired: by using `simple-phpunit` from MakerBundle, PHPUnit *was* being downloaded fresh on every run. That's just a guess, but if I'm right, it would make a significant difference